controller_manager/test/controller_manager_test_common.hpp

```diff
@@ -63,7 +63,7 @@ class ControllerManagerFixture : public ::testing::Test
     updater_ = std::thread([&](void) -> void {
       while (run_updater_)
       {
-        cm_->update();
+        cm_->update(rclcpp::Duration::from_nanoseconds(100000000));
```

Comment:
> Should we provide dt or absolute current time

I didn't see the discussion when I started, so I went with dt, but I can see why time would make sense, especially if there's a plan to have different controllers execute at different rates.

What I'm not sure about is what would be the source of the absolute time, would it come from ros, or something like steady_clock, or something else?

Either way, I'm happy to try and implement the change if you'd like to proceed this way.

> Anyway, IMO, we should provide read measured time since the last update call and not theoretical one.

I agree. But I wasn't sure about the right source of time to use, so I just used the theoretical update rate.

Also, what would we provide on the very first iteration?

> I think also that the following is possible: (it would make more sense and less "0")

Yup, agreed. I'll change that too depending on what you'd like to do about point 1.

Reply:
will do.

Reply:
done.